documents/plan.md

```diff
@@ -1,11 +1,15 @@
 # Project plan
 
 ## Language evaluation
+At a bare minimum, the DSL will need to be able to load and unload data to and from something and perform basic math operations on all elements in the vector. However, I expect that any decent implementation will have a lot more to it than that and will have more of the "fancy" features outlined in the description. Example code can be used to verify functionality (that the DSL does what it should using the given syntax) and benchmarks can verify that we are getting a performance increase over a standard implementation. I figure I will also get feedback on my syntax from my critique group as well.
```

Comment:
What does it mean to "load and unload data to and from something"? What is the something -- disk? arrays?